### ISSUE TITLE
Unused variable false positive with function forward

### ISSUE THREAD

**Issue by movedoa** [Riverside-Software/sonar-openedge#61]
(The content of a variable is neither read nor written during the flow of execution)

Prototype definitions of functions should be excluded from the rule.
At the moment, all parameters are marked as usused.
```
FUNCTION FooFunction RETURNS LOGICAL(
   cFooParam1 AS CHARACTER, 
   dFooParam2 AS DECIMAL
)  FORWARD.
```

**Comment by gquerret** [comment]
Fixed in development.
Super-procedures, external procedures, forward declaration, and `IN` procedure declaration are ignored.
